source-map-support.js

```diff
@@ -82,8 +82,15 @@ function retrieveSourceMapURL(source) {
   // picking up sourceMappingURLs from comments, strings, etc.
   var lastMatch, match;
   while (match = re.exec(fileData)) lastMatch = match;
-  if (!lastMatch) return null;
-  return lastMatch[1];
+  if (lastMatch) {
+    return lastMatch[1];
+  } else {
+    // Try the .map file
+    mapFile = source + '.map';
+    if (fs.existsSync(mapFile)) {
```

Comment:
@evanw is the browser support even necessary?

I can only see this useful in node.js setup. As long as it does not break Browser, it would be fine.